### ISSUE TITLE
Drop garment features

### ISSUE THREAD

**Issue by baopmessi** [ShineChen1024/MagicClothing#74]
Thanks for awesome work!. I wonder howu to drop garment feature during training ? In classifier free guidance, I see you set ref hidden_states=0. When implementing the training script, I also set attn_score=0 with drop rate of 0.05. but if you do so, ref_unet will not be able to go backward and get the gradient ? because ref_unet is not involved in calculating pred_noise. Hope you answer my questions
 
![image](https://github.com/ShineChen1024/MagicClothing/assets/34626618/d0f87041-65dd-46a0-8fc1-b715ea2a28ae)

**Comment by xinli2008** [comment]
have you soloved this question?

**Comment by ShineChen1024** [comment]
torch.zeros_like
